channels/constants.js

```diff
@@ -2,5 +2,5 @@ module.exports = {
   CHANNEL_TYPE_PRIMARY_TRACK: 'primary-track',
   CHANNEL_TYPE_SAMPLE_TRACK: 'sample-track',
   CHANNEL_TYPE_TRANSITION: 'transition',
-  CHANNEL_TYPE_MIX: 'mix',
+  CHANNEL_TYPE_MIX: 'mix'
```

Comment:
ah yes. i'm actually with you on trailing commas, but for now i do `standard`. it's possible to use [`standard-engine`](https://github.com/flet/standard-engine) to make your own.